SETUP.md

```diff
@@ -9,6 +9,10 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 
 1. Request and Configure AWS access (code.org staff) or configure local secrets (open source contributors). See [Configure AWS Access or Secrets](#configure-aws-access-or-secrets) below. This step is not required until rake is first run below, but staff may wish to submit the request first so its ready when rake is.
 
+1. Clone the repo, which also may take a while. `git clone git@github.com:code-dot-org/code-dot-org.git`
```

Comment:
Should we keep the `https` instructions from previously?  I always end up using that.

(Also: also?)

Reply:
I'll chime in since IIRC I originally edited this line to recommend SSH above HTTPS. The reason was because SSH (for me at the time) allowed me to not have to type in my password every time I push to github. With HTTPS @breville are you having to type in your password every time you push? If you are, then I'd say this option is safe to remove from our setup instructions, and you can always find the HTTPS clone instructions on the github page for our repo. If not... I still think I agree with Darin that having only one option here is sufficient, unless it can be shown that there are unique advantages to both approaches.

Reply:
IMO this is Git usage knowledge that doesn't apply to our org in any unique way. If there was a reason to encourage HTTPS connections in certain scenarios, I'd rather way what that scenario is and leave how to do it as an exercise for the reader.

Reply:
My understanding is that HTTP is optimal for the initial clone, since downloads are faster over HTTP and our repository is huge. But then after that first clone, SSH is optimal for easy reauthentication

Reply:
Fair points!

Reply:
Oh... that's interesting! Somebody want to take a stab at suggesting a new phrasing here? Otherwise I'll circle back after meetings.

Reply:
quite interesting, nice find Elijah! In that case, one idea would be to break this into 2 steps:
1. HTTPS clone 
2. [switch repo url to SSH](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories#changing-a-remote-repositorys-url)

Reply:
Maybe something like,

```suggestion
1. Clone the repo, which also may take a while.
  - The simplest option is to clone via SSH with: `git clone git@github.com:code-dot-org/code-dot-org.git`
  - The fastest option is to clone via HTTP with: `git clone https://github.com/code-dot-org/code-dot-org.git`. Although faster than SSH, this option requires you to reauthenticate every time you want to update. You will therefore probably want to switch to SSH after the initial clone with `git remote set-url origin git@github.com:code-dot-org/code-dot-org.git`
```